openmetadata-spec/src/main/resources/json/schema/api/data/createGlossary.json

```diff
@@ -10,7 +10,10 @@
   "properties": {
     "name": {
       "description": "Name that identifies this glossary.",
-      "$ref": "../../type/basic.json#/definitions/entityName"
+      "type": "string",
```

Comment:
@07Himank lets change the entityName itself

Reply:
done in latest commit